demo-app/src/pages/aruba.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Related to my directory structure comment earlier, what about the following which will keep all related files contained?

/pages
|__ /network-devices
&nbsp;&nbsp;|__ /data
  &nbsp;&nbsp;&nbsp;&nbsp;|__ network-device-data.js  
&nbsp;&nbsp;|__ index.jsx // actual page
&nbsp;&nbsp;|__ NetworkMap.jsx
&nbsp;&nbsp;|__ NetworkPin.jsx
&nbsp;&nbsp;

Reply:
Done